rpcs3/Emu/Cell/SPUThread.h

```diff
@@ -764,6 +764,8 @@ class spu_thread : public cpu_thread
 	const char* current_func{}; // Current STOP or RDCH blocking function
 	u64 start_time{}; // Starting time of STOP or RDCH bloking function
 
+	atomic_t<u8> debugger_float_mode = 0;
```

Comment:
Hmm, why does it have to be in thread context?

Reply:
1) better be thread-specific.
2) detectable by new thread context change detection, the memcmp will fail and update it automatically. (instead of doing nothing)